ui/src/components/expandable_container/ExpandableDescriptionList.scss

```diff
@@ -2,7 +2,7 @@
 
 .expandableContainer__childWrapper {
   height: 0;
-  overflow-y: auto;
+  overflow: hidden;
```

Comment:
Without this, we see unnecessary scroll:

![Screenshot 2020-12-03 at 12 32 17 PM](https://user-images.githubusercontent.com/23465343/100964744-39f08d80-3564-11eb-9277-fe60fc2cb993.png)

![Screenshot 2020-12-03 at 12 30 53 PM](https://user-images.githubusercontent.com/23465343/100964754-3f4dd800-3564-11eb-90fa-8799c84c36be.png)

Reply:
Yeah the second screenshot is from the Docker Env Variables. If you want to see the scroll there (I thought it was weird to have that and the Expand option), I will apply this style change only for the activity log's expandable container.

Reply:
Now the scroll behavior is controlled by the presence of the class `scrollableContainer`.